packages/beacon-node/src/util/workerEvents.ts

```diff
@@ -1,9 +1,14 @@
 import {MessagePort, Worker} from "node:worker_threads";
+import {Metrics} from "../metrics/metrics.js";
+import {NetworkCoreWorkerMetrics} from "../network/core/metrics.js";
 import {StrictEventEmitterSingleArg} from "./strictEvents.js";
 
+const nanoToSecondConversion = 1e9;
```

Comment:
Just minor remark, we use CONSTANT_CASE for such values in most parts of the code base

For example

https://github.com/ChainSafe/lodestar/blob/e690ac6df8b6c3e766fd5c245254575dad061f7b/packages/cli/src/cmds/beacon/handler.ts#L29